src/server/utils/formRunnerSession.ts

```diff
@@ -0,0 +1,34 @@
+import { createFormRunnerEditListItemLink, createFormRunnerReturningUserLink } from "server/components/lists/helpers";
+import { generateFormRunnerWebhookData, getNewSessionWebhookData } from "server/components/formRunner/helpers";
+import { getInitiateFormRunnerSessionToken } from "server/components/dashboard/helpers";
+import type { BaseListItemGetObject, List } from "server/models/types";
+
+interface InitaliseFormRunnerInput {
+  list: List;
+  listItem: BaseListItemGetObject;
+  message: string;
+  isUnderTest: boolean;
+  isAnnualReview?: boolean;
+}
+
+export default async function initialiseFormRunnerSession({
```

Comment:
utils are typically very small and very flexible. think about the same size as lodash functions. typically with very few dependencies on other functions and can be used almost anywhere.

this can be a helper - it can only be used in certain places, does a fairly specific thing.

I'd also think about the "distance" from the other related functions. All of the imports in this file are at `server/components/formRunner/helpers` or `server/components/lists/helpers`, and now there is a new one, `src/server/utils`. 

you might find it useful to use a diagram/visualisation for refactors. there is one built into jetbrains using `yFiles`. probably similar plugin for vsc.
![image](https://user-images.githubusercontent.com/22080510/211049478-816339c6-5fcc-40c1-a33f-ea28eb6b6985.png)

where it was, in listsItemsControllers probably wasn't the best choice to begin with so I see the motivation to move it.

this is probably more likely to be a service than a util, because I can see that there are other functions relating to formRunner. 

```
class FormRunnerService {
  constructor(options) {
    // maybe things like 
    this.url = options.url;
    this.isUnderTest = options.isUnderTest //which preferably we don't have this at all.. anyway.. 
  }

  generateWebhookData(..) {..}

  createEditListItemUrl(..) {..}

  initialiseSession(..) {..}

  //etc.
  

}

// alternatively, if there is no need for initialisation

 export function generateWebhookData(..) {..}
 export function createEditListItemUrl(..) {..}
 export function initialiseSession(..) {..}


// in controllers.ts
import * as formRunnerService from 'server/services';

formRunnerService.generateWebhookData(..)



```